### PR TITLE
[ServerManagement] Support for 2016-07-01-preview updates

### PR DESCRIPTION
**Pull request by r22016**
created a new version of swagger spec that includes changes to support the latest preview api updates

testing:
- used autorest to generate C# sdk on local machine
- built new version of sdk and verified existing tests pass
- updated tests to include calls to service updates, verified that these work

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@r22016__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (rameshr). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: arm-servermanagement/2016-07-01-preview/servermanagement.json [hunk 1]
```diff
@@ -0,0 +1,2234 @@
+{
```
Comment (amarzavery) [inline arm-servermanagement/2016-07-01-preview/servermanagement.json]: indentation 2 spaces.
If using VS  Ctrl + A and then Ctrl + K + D.
Make sure that tab are set to 2 spaces and are also replaced by spaces in Tools -> Options settings for json documents.

**Comment by amarzavery** [comment]
@fearthecowboy - you should definitely review this.

**Comment by fearthecowboy** [comment]
thanks @amarzavery  .. on it!

File: arm-servermanagement/2016-07-01-preview/servermanagement.json [hunk 2]
```diff
@@ -0,0 +1,2234 @@
...
+                        "description": "The PowerShell sessionId from the user"
+                    },
+                    {
+                        "name": "PowerShellTabCompletionParamters",
+                        "x-ms-client-flatten": true,
+                        "description": "Parameters supplied to the tab completion call.",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/
```
Comment (fearthecowboy) [inline arm-servermanagement/2016-07-01-preview/servermanagement.json]: probably should have ` "format": "date-time", ` on the `publishedTypeUtc

Comment (fearthecowboy) [inline arm-servermanagement/2016-07-01-preview/servermanagement.json]: Given the berevity in the names, there should really be descriptions on these.

Comment (fearthecowboy) [inline arm-servermanagement/2016-07-01-preview/servermanagement.json]: Did you get the Visual Studio Code linter working ? The missing descriptions should have been caught by the editor.

**Review by fearthecowboy — CHANGES_REQUESTED** [review]
You should install the VsCode and the OpenAPI (swagger) linter. 

Just download 

https://github.com/Azure/openapi-lint-extension/releases/download/v0.5-pre/swagger-linter-0.5.0.vsix 

and open that file in VS Code (and restart when it tells you). No other config should be needed.

You'll see the missing description notifications (there are only 15 missing descriptions in the whole doc)

**Review by fearthecowboy — APPROVED** [review]

**Comment by fearthecowboy** [comment]
Looks good to me; once the CI is done I'll merge it.

**Pull request merged**